src/spikeinterface/core/recording_tools.py

```diff
@@ -149,6 +149,38 @@ def get_closest_channels(recording, channel_ids=None, num_channels=None):
     return np.array(closest_channels_inds), np.array(dists)
 
 
+def get_noise_levels(recording: 'BaseRecording', return_scaled: bool = True, method: Literal['mad', 'std'] = "mad", 
+    force_recompute: bool=False, **random_chunk_kwargs):
+    """
+    Estimate noise for each channel using MAD methods.
+    You can use standard deviation with `method='std'`
+
+    Internally it samples some chunk across segment.
+    And then, it use MAD estimator (more robust than STD)
+
+    """
+    
+    if 'noise_levels' in recording._properties and not force_recompute:
+        noise_levels = recording.get_property(key="noise_levels")
+        return noise_levels
+    else:
+        random_chunks = get_random_data_chunks(
+            recording, return_scaled=return_scaled, **random_chunk_kwargs
+        )
+        
+        if method == "mad":
+            med = np.median(random_chunks, axis=0, keepdims=True)
+            # hard-coded so that core doesn't depend on scipy
+            noise_levels = (
+                np.median(np.abs(random_chunks - med), axis=0) / 0.6744897501960817
+            )
+        elif method == "std":
+            noise_levels = np.std(random_chunks, axis=0)
+        recording.set_property('noise_levels', noise_levels)
```

Comment:
I would call the property `noise_level`, no s

Reply:
Done